components/raytac/MDBT50Q.stanza

```diff
@@ -0,0 +1,211 @@
+  manufacturer = "Raytac"
+  mpn = switch(antenna):
+    "chip" : "MDBT50Q-512K"
+    "pcb"  : "MDBT50Q-P512K"
+  description = "nRF52833 based module, with chip or PCB antenna"
+
+  pin-properties :
+    [pin:Ref | pads:Int ...    | side:Dir ]
+    [GND    | 1, 2, 15, 33, 55 | Down ]
+    [NC[0]  | 4                | Up   ]
+    [NC[1]  | 31               | Up   ]
+    [NC[2]  | 42               | Up   ]
+    [NC[3]  | 45               | Up   ]
+    [NC[4]  | 49               | Up   ]
+    [NC[5]  | 59               | Up   ]
+    [NC[6]  | 60               | Up   ]
+    [P0_25  | 3                | Left ]
+    [P0_23  | 5                | Left ]
+    [P1_05  | 6                | Left ]
+    [P1_03  | 7                | Left ]
+    [P0_19  | 8                | Left ]
+    [P0_03  | 9                | Left ]
+    [P0_29  | 10               | Left ]
+    [P0_02  | 11               | Left ]
+    [P0_31  | 12               | Left ]
+    [P0_28  | 13               | Left ]
+    [P0_30  | 14               | Left ]
+    [P0_27  | 16               | Left ]
+    [P0_00  | 17               | Left ]
+    [P0_01  | 18               | Left ]
+    [P0_26  | 19               | Left ]
+    [P0_04  | 20               | Left ]
+    [P0_05  | 21               | Left ]
+    [P0_06  | 22               | Left ]
+    [P0_07  | 23               | Left ]
+    [P0_08  | 24               | Left ]
+    [P1_08  | 25               | Left ]
+    [P1_09  | 26               | Left ]
+    [P0_11  | 27               | Left ]
+    [VDD    | 28               | Right]
+    [P0_12  | 29               | Left ]
+    [VDDH   | 30               | Right]
+    [VBUS   | 32               | Right]
+    [D-     | 34               | Right]
+    [D+     | 35               | Right]
+    [P0_14  | 36               | Left ]
+    [P0_13  | 37               | Left ]
+    [P0_16  | 38               | Left ]
+    [P0_15  | 39               | Left ]
+    [P0_18  | 40               | Left ]
+    [P0_17  | 41               | Left ]
+    [P0_21  | 43               | Left ]
+    [P0_20  | 44               | Left ]
+    [P0_22  | 46               | Left ]
+    [P1_00  | 47               | Left ]
+    [P0_24  | 48               | Left ]
+    [P1_02  | 50               | Left ]
+    [SWDIO  | 51               | Right]
+    [P0_09  | 52               | Left ]
+    [SWDCK  | 53               | Right]
+    [P0_10  | 54               | Left ]
+    [P1_04  | 56               | Left ]
+    [P1_06  | 57               | Left ]
+    [P1_07  | 58               | Left ]
+    [P1_01  | 61               | Left ]
+
+  make-box-symbol()
+  assign-landpattern(module-pkg)
+
+pcb-landpattern module-pkg:
+  val padx = 0.6
+  val pady = 0.4
+  val pitch = 0.8
+  val offset = 0.9
+
+  pad p[1] : smd-pad(padx, pady) at loc(0.6, 11.5)
+  pad p[2] : smd-pad(padx, pady) at loc(0.6, 10.4)
+  pad p[3] : smd-pad(padx, pady) at loc(0.6, 9.6)
+  for (i in 4 to 15) do :
+    val div = to-double(i % 2)
+    if (div > 0.0) :
```

Comment:
Coding style argument? Stanza doesn't require the parentheses. We should probably keep OCDB consistent.